vsts/build-test-nuget.yml

```diff
@@ -43,49 +43,10 @@ jobs:
           ArtifactName: unsigned_nuget_packages
         condition: always()
 
-      - task: PowerShell@2
```

Comment:
Moved partner upload to a separate task, added variable to turn on and off E2E tests in the case we want to run E2E tests separately and run the build and publish only setting.